Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugRoadRegen.java

```diff
@@ -18,20 +16,16 @@
 
     @Override public boolean onCommand(PlotPlayer player, String[] args) {
         Location loc = player.getLocation();
-        PlotArea plotArea = loc.getPlotArea();
-        if (!(plotArea instanceof HybridPlotWorld)) {
-            return sendMessage(player, Captions.NOT_IN_PLOT_WORLD);
-        }
+        PlotManager manager = loc.getPlotArea().getPlotManager();
```

Comment:
Isn't it likely that this may cause a NPE since not every location is going to contain a plot area?

Reply:
Ah, a detail I had not considered. Should be fixed with my latest commit.